src/docs/sphinx/advancedExamples/validationStudies/faultMechanics/faultVerification/Example.rst

```diff
@@ -2,7 +2,7 @@
 
 
 ###########################################################
-Verification of Induced Stresses Along a Fault
+Verification of Induced Sresses Along a Fault
```

Comment:
```suggestion
Verification of Induced Stresses Along a Fault
```

Reply:
My bad, this change was not intentional. It is fixed now.